src/Resources/skeleton/doctrine/Entity-use-composition.tpl.php

```diff
@@ -0,0 +1,40 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace ?>;
+
+<?php if ($api_resource): ?>use ApiPlatform\Core\Annotation\ApiResource;
+<?php endif ?>
+use <?= $repository_full_class_name ?>;
+use Doctrine\ORM\Mapping as ORM;
+<?php if ($broadcast): ?>use Symfony\UX\Turbo\Attribute\Broadcast;
+<?php endif ?>
+
+/**
+<?php if ($api_resource && !$use_attributes): ?> * @ApiResource()
+<?php endif ?>
+<?php if ($broadcast && !$use_attributes): ?> * @Broadcast()
+<?php endif ?>
+ * @ORM\Entity
+<?php if ($should_escape_table_name): ?> * @ORM\Table(name="`<?= $table_name ?>`")
+<?php endif ?>
+ */
+<?php if ($api_resource && $use_attributes): ?>
+#[ApiResource]
+<?php endif ?>
+<?php if ($broadcast && $use_attributes): ?>
+#[Broadcast]
+<?php endif ?>
+class <?= $class_name."\n" ?>
```

Comment:
We should be able to use the existing `doctrine/Entity.tpl.php` template. The only difference that I see is this tpl doesnt include the `@ORM\Entity(repositoryClass=.....` docBlock. We can accomplish the same in the existing template by adding a conditional

Reply:
I though about that... but having one single template for both use cases will be a bit messy don't you think ?

Reply:
Ok I did it, I tried to be as clean as possible.

The hardest task was to comply with the empty lines and spacing.

In the end, only one line of the reference generated file (fixture in the test) is modified.
The modification is about adding the return type `:array`.

Reply:
I fixed that, thanks for the suggestion.